Sources/TokamakStaticHTML/Modifiers/LayoutModifiers.swift

```diff
@@ -101,3 +101,11 @@ extension _PaddingLayout: DOMViewModifier {
       .joined(separator: " ")]
   }
 }
+
+extension _ShadowLayout: DOMViewModifier {
+  public var attributes: [HTMLAttribute: String] {
+    ["style": "box-shadow: \(x)px \(y)px \(radius)px 0px \(color.cssValue(.defaultEnvironment));"]
```

Comment:
Maybe you can multiply the radius by 2? Seems like when going from [Sketch to UIKit](https://sarunw.com/posts/match-view-shadow-to-sketch-shadow/#blur-vs.-shadow-radius) (which uses blur and spread), they'll divide by two, so the inverse could maybe work here?

Reply:
Yes, that's at least closer than what I had (color meter says it's not a match, but it's close enough for my naked eye)